### ISSUE TITLE
[Android] Image.getSize doesn't work with images from hybrid app's resources

### ISSUE THREAD

**Issue by hwth** [facebook/react-native#23816]
## 🐛 Bug Report
  I use Image.getSize to get width, height of images from hybrid app's resources (Drawable folder on Android and Images.xcassets on Xcode). It works well with iOS but I got failed with Android.

Can anyone fix it or suggest a way to get size of images from native.
Thanks
## Code Example
` Image.getSize('background', (width, height) => {
      // do something
});`

## Environment
 React Native Environment Info:

    System:
      OS: macOS 10.14.3
      CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
      Memory: 197.98 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh

    Binaries:
      Node: 11.5.0 - /usr/local/bin/node
      Yarn: 1.9.4 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman

    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 23, 24, 25, 26, 27, 28
        Build Tools: 23.0.1, 25.0.1, 25.0.2, 25.0.3, 26.0.1, 26.0.2, 26.0.3, 27.0.3, 28.0.2, 28.0.3

    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild

    npmPackages:
      react: 16.6.3 => 16.6.3
      react-native: 0.58.5 => 0.58.5

    npmGlobalPackages:
      react-native-cli: 2.0.1

**Comment by hwth** [comment]
@react-native-bot done!

**Comment by maxiee** [comment]
I was able to reproduce this issue with code:

```jsx
import React, {Component} from 'react';
import { View, Image } from 'react-native';

export default () => {
    console.log("maxieeee :" + JSON.stringify(Image.getSize('react_native')))

    return <View>
        <Image 
            source={{uri: 'react_native'}} 
            style={{
                width: 200, 
                height: 150,
                borderWidth: 1,
                borderColor: '#999999'
            }}/>
    </View>
}
```

The screenshot is:

![device-2019-03-08-194540](https://user-images.githubusercontent.com/3392007/54027135-574fc680-41db-11e9-8099-4a2f8e677898.png)

react-native info:

```
  React Native Environment Info:
    System:
      OS: macOS 10.14.3
      CPU: (8) x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
      Memory: 254.92 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 11.8.0 - /usr/local/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      Watchman: 4.7.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 19, 21, 22, 23, 24, 25, 26, 27, 28
        Build Tools: 19.1.0, 21.1.2, 22.0.1, 23.0.1, 23.0.2, 23.0.3, 24.0.0, 24.0.1, 24.0.2, 25.0.0, 25.0.1, 25.0.2, 25.0.3, 26.0.1, 26.0.2, 26.0.3, 27.0.0, 27.0.1, 27.0.2, 27.0.3, 28.0.2, 28.0.3
        System Images: android-23 | Android TV ARM EABI v7a, android-23 | Android TV Intel x86 Atom, android-23 | Android Wear ARM EABI v7a, android-23 | Google APIs ARM EABI v7a, android-23 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5314842
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.3 => 16.6.3 
      react-native: 0.58.6 => 0.58.6 
```

I have a look into the source (Libraries/Image/Image.android.js). 

I Found that `ImageLoader.getSize(url)` throws exception and goes into catch block (Libraries/Image/Image.android.js:138).

**Comment by hwth** [comment]
Is it fixed in 0.59.1?
